pkg/machine/shim/host.go

```diff
@@ -118,6 +118,19 @@ func Init(opts machineDefine.InitOptions, mp vmconfigs.VMProvider) error {
 		createOpts.UserModeNetworking = *umn
 	}
 
+	// Mounts
+	if mp.VMType() != machineDefine.WSLVirt {
+		mc.Mounts = CmdLineVolumesToMounts(opts.Volumes, mp.MountType())
+	}
+
+	for _, mnt := range mc.Mounts {
+		// Issue #18230 ... cannot mount over /tmp
+		mountTarget, _ := strings.CutSuffix(mnt.Target, "/")
```

Comment:
I have not checked how our path mounting handles non sanitized inputs but `//tmp` `/tmp//` etc are all the same path. The way to clean them is using `path.Clean()`. Note *path* not *filepath* as filepath on windows would use `\` but as this is the target path it must be using `/`